runtime/compiler/control/JITaaSCompilationThread.cpp

```diff
@@ -568,6 +568,7 @@ bool handleServerMessage(JITServer::ClientStream *client, TR_J9VM *fe)
          vmInfo._writeBarrierType = TR::Compiler->om.writeBarrierType();
          vmInfo._compressObjectReferences = TR::Compiler->om.compressObjectReferences();
          vmInfo._processorFeatureFlags = TR::Compiler->target.cpu.getProcessorFeatureFlags();
+         vmInfo._invokeWithArgumentsHelperMethod = J9VMJAVALANGINVOKEMETHODHANDLE_INVOKEWITHARGUMENTSHELPER_METHOD(fe->getJ9JITConfig()->javaVM);
```

Comment:
I couldn't find where this macro is defined, for some reason. But it looks like this information should not change at runtime.

Reply:
Liqun says it's in `oti/j9vmconstantpool.h`
```
#define J9VMCONSTANTPOOL_JAVALANGINVOKEMETHODHANDLE_INVOKEWITHARGUMENTSHELPER 293
#define J9VMJAVALANGINVOKEMETHODHANDLE_INVOKEWITHARGUMENTSHELPER_METHOD(vm) J9VMCONSTANTPOOL_STATICMETHOD_AT(vm, J9VMCONSTANTPOOL_JAVALANGINVOKEMETHODHANDLE_INVOKEWITHARGUMENTSHELPER)
```
and it points to `private static native Object invokeWithArgumentsHelper(MethodHandle mh, Object[] args);`

Reply:
Somehow, my openj9 repo doesn't have this file.
But it seems that this helper cannot be changed during runtime, so we should be good.

Reply:
It gets generated automatically. After I compiled using "make all" it appeared in
`./build/linux-x86_64-normal-server-release/vm/oti/j9vmconstantpool.h`